packages/core/demo/data/line.ts

```diff
@@ -39,6 +72,42 @@ export const lineOptions = {
 	}
 };
 
+export const sparklineLoadingOptions = {
+	title: "Sparkline (loading)",
+	axes: {
+		bottom: {
+			visible: false,
+			title: "2019 Annual Sales Figures",
+			mapsTo: "date",
+			scaleType: "time"
+		},
+		left: {
+			visible: false,
+			mapsTo: "value",
+			title: "Conversion rate",
+			scaleType: "linear"
+		}
+	},
+	grid: {
+		x: {
+			enabled: false
+		},
+		y: {
+			enabled: false
+		}
+	},
+	legend: {
+		enabled: false
+	},
+	tooltip: {
+		enabled: false
+	},
+	points: {
+		enabled: false
+	},
+	sparklineLoading: true
```

Comment:
can this work the same way as other components? Essentially you should be using the same loading flag here as a bar or line chart & it would load up the correct loading state based on the chart type.

Reply:
`sparklineLoading` option is now being set within `data`. Just like how we set the loading state for other types of chart.